docker/1.4.4/config/mosquitto.conf.default

```diff
@@ -0,0 +1,819 @@
+# Config file for mosquitto
+#
```

Comment:
I guess it doesn't need to be here at all. My intent was to have an example config that could be changed if needed. I will remove these files.